src/App.js

```diff
@@ -1,13 +1,16 @@
+import React from 'react';
+// eslint-disable-next-line import/extensions
```

Comment:
- [ ] Kindly remove this eslint disable line it's best to fix the issue in the code.